docs/userGuide/syntax/navBars.mbdf

```diff
@@ -69,13 +73,42 @@ If you wish to further customize your navbar beyond the primary, dark, and light
 <md>In `{your-site}/_markbind/headers/header.md`, you can change `<navbar type="dark/primary/light">` to `<navbar type="none" add-class="bg-warning/danger/info/primary/success">` to apply Bootstrap background styles.</md>
 </modal>
 
+**Navbars let you specify rules to customise link highlighting.**
```

Comment:
Let's move this above the summary options table, since you've already mentioned `default-highlight-on` here